### DISCUSSION TITLE
📢  Join Us for a Live Talk with Dr. Lu Li on His Antarctic 🐧Expedition! 📢

### DISCUSSION THREAD

**Discussion by MGomezN** [fatiando/community#152] [Announcements]
In this talk, Dr. @LL-Geo , a valued team member of Fatiando a Terra, provides fascinating insights into the science and daily life of the Denman Terrestrial Campaign. Discover the challenges and triumphs of conducting research in one of the most remote and extreme environments on Earth.

🗓 Date: Tuesday, May 21st
🕒 Time: 14:00 UTC
:calling: Join here: [https://meet.jit.si/fatiando-a-terra](https://meet.jit.si/fatiando-a-terra) 
📺 [Live Streaming on YouTube](https://youtube.com/live/XNJ1WRfaSkk?feature=share)

**Comment by mdtanker** [comment]
Any chance this can be recorded? I'd love to watch, but unfortunately can't attend then :face_exhaling:

**Reply by MGomezN** [reply]
Of course! The talk will be available on Youtube to be watched asynchronously
